_posts/pl/2022-01-11-smarthome.md

```diff
@@ -0,0 +1,316 @@
+Ciekawostką może być to, że sensory Bluetooth Low Energy, póki są niepołączone z konkretnym urządzeniem, nadają informacje o swoim stanie w eter w taki sposób,
+że odczytać może je każdy. Stąd odbieram dane z zegarka Xiaomi sąsiadów. Z urządzeń Bluetooth mam następujące:
+
+#### Xiaomi Flora flower monitor
+
+![Czujnik do roślin](/assets/img/posts/2022-01-11-smarthome/flowermonitor.png){: width="250" }
+
+Zalety — duża liczba danych, zarówno o glebie, jak i otoczeniu. Z danych otoczenia mamy np. temperature oraz jasność.
+
+Wady — to na pewno komunikacja bluetooth i to, co za sobą ciągnie.
+
+#### Zegarek z wyświetlaczem eInk (lywsd02mmc)
+
+![Zegar z wyświetlaczem eInk](/assets/img/posts/2022-01-11-smarthome/einkclock.png){: width="250" }
+
+Zalety — na pewno czytelny wyświetlacz i dodatkowe dane o wilgotności oraz temperaturze w pomieszczeniu.
+
+Wady — takie jak każdego urządzenia Bluetooth (m.in zasięg).
+
+### Komunikacja po ZigBee
+
+Mój ulubiony sposób komunikacji, bo najmniej zawodny oraz łatwo zwiększa się zasięg.
+Dodatkowo są to urządzenia, które bardzo długo wytrzymują na baterii.
+Urządzeń o tym sposobie komunikacji mam najwięcej, wiec zbiorę je w kilku grupach.
+
+#### Przekaźniki oraz gniazdka
+
+Pozwoliłem sobie zebrać te wszystkie urządzenia w jedną grupę, bo ich głównym przeznaczeniem jest włączenie/wyłączenie zasilania. 
+Różnią się jedynie wspieranymi funkcjami. Do których można zaliczyć wybór typu klawisza (w przypadku przekaźników), 
+liczbę i rodzaj raportowanych danych. Wybrane informują o poborze energii, temperaturze, a także pozwalają ustawić stan po powrocie zasilania. 
+Warto zwrócić też uwagę na to, aby mieściły się w europejskich puszkach (jeśli są to przekaźniki).
+Tak jak wcześniej już wspomniałem, zaletą tych urządzeń jest to, że większość z nich wzmacnia zasięg sieci ZigBee.
+
+![Przekaźnik](/assets/img/posts/2022-01-11-smarthome/relay.png){: width="250" }
+![Gniazdko](/assets/img/posts/2022-01-11-smarthome/socket.png){: width="250" }
+
+
+#### Sterowniki taśm led
+
+Sterowniki taśm led, czyli urządzenia, które pozwalają nam na wybór koloru oraz jasności.
+W swoim SmartHome mam tylko jedno takie urządzenie, które działa prawie bez zastrzeżeń.
+Jedyną wadą jest to, że po powrocie zasilania jest domyślnie włączone i musiałem to „ograć” dodatkowym przekaźnikiem i automatyzacją.
+Z zalet — podłączamy dowolną taśmę LED oraz dowolny zasilacz (oczywiście dobieramy urządzenia w zależności od potrzebnej mocy i napięcia).
+W moim wypadku jest to taśma LED zasilona z 24V na dwóch końcach (dla równomiernego podświetlenia, w przypadku większych długości).
+
+![Sterownik taśmy LED](/assets/img/posts/2022-01-11-smarthome/ledcontroller.png){: width="250" }
+
+
+#### Przyciski
+
+W przypadku przycisków podobnie jak i w przypadku przekaźników jest w czym przebierać. 
+Szczególnie lubię te najmniejsze, z dwoma przyciskami pochodzące z rodziny IKEA Tradfri. 
```

Comment:
Szczególnie lubię te najmniejsze, _zwłaszcza dwa przyciski_ pochodzące z rodziny IKEA Tradfri.

Reply:
to jest jedno urządzenie posiadające dwa przyciski (na załączonych zdjeciach jest to ten pierwszy)

Reply:
ok, to może:
_Szczególnie lubię te najmniejsze urządzenia z dwoma przyciskami, pochodzące z rodziny IKEA Tradfri._ 
a jak jeszcze nie to, to może zostać tak jak było :)